src/lib/addons/slack.test.ts

```diff
@@ -44,252 +46,298 @@ jest.mock(
                     retries,
                     backoff,
                 });
-                return Promise.resolve({ status: 200 });
+                return Promise.resolve({ ok: true, status: 200 });
             }
 
-            async registerEvent(_) {
-                return Promise.resolve();
+            async registerEvent(event) {
+                return registerEventMock(event);
             }
         },
 );
 
-test('Should call slack webhook', async () => {
```

Comment:
Adding the `describe` messed up the diff, but the only real change is the new `Should call registerEvent` test.